runtime/doc/news.txt

```diff
@@ -125,6 +125,10 @@ The following changes may require adaptations in user config or plugins.
 • Returning any truthy value from a callback passed to |nvim_create_autocmd()|
   (rather than just `true`) will delete the autocommand.
 
+• |vim.lsp.util.extract_completion_items()| will no longer return reliable
```

Comment:
This could also mention the added support for itemDefaults and maybe expand on the implications (lower completion overhead for servers supporting it, and that it could also break 3rd party plugins which don't support it - maybe with a hint how to remove the capability)